docs/user_manual_how_to_use_1_operator_mode_data_editor_tab.rst

```diff
@@ -66,7 +66,7 @@ salinity profiles drawn in solid blue (left to right, respectively, in :numref:`
 During the import stage, the geographic position and date in the input file are used to query WOA or RTOFS
 atlases (if available) to obtain mean sound speed, temperature and salinity profiles (to provide a context
 during data editing), which are drawn in dashed colours. Thus, it is important that the cast positional metadata
-are correct for this lookup operation.
+is correct for this lookup operation.
```

Comment:
https://english.stackexchange.com/questions/180801/is-metadata-considered-plural